README.MD

```diff
@@ -1,4 +1,7 @@
 # SonarLint for Visual Studio
+
+[![Waffle.io - Columns and their card count](https://badge.waffle.io/SonarSource/sonar-csharp.svg?columns=all)](https://waffle.io/SonarSource/sonar-csharp)
```

Comment:
Perhaps rename link to "Waffle board"? Columns and card cound sounds odd.

Reply:
It just displays 'waffle'